api/src/main/java/com/netflix/iceberg/RewriteFiles.java

```diff
@@ -30,4 +32,5 @@
  * will throw a {@link CommitFailedException}.
  */
 public interface RewriteFiles extends PendingUpdate<Snapshot> {
+    RewriteFiles replaceFiles(Set<DataFile> filesToDelete, Set<DataFile> filesToAdd);
```

Comment:
This should have Javadocs.

Also, it seems odd to use both "replace" and "rewrite" in this API. Let's standardize on just "rewrite". `ReplaceFiles` can still be the internal implementation.

Reply:
Changed and javadocs added.